app/scripts/metamask-controller.js

```diff
@@ -5581,7 +5606,11 @@ export default class MetamaskController extends EventEmitter {
    */
   set isClientOpen(open) {
     this._isClientOpen = open;
-    this.detectTokensController.isOpen = open;
+    if (open) {
+      this.tokenDetectionController.enable();
+    } else {
+      this.tokenDetectionController.disable();
+    }
```

Comment:
`TokenDetectionController` no longer has an assignable `isOpen` property. 

Instead, it has a `disable` class field that blocks all network requests, and can be toggled with the public methods `enable`, `disable`.